test/integration/rules/aria-form-field-label/aria-form-field-label.html

```diff
@@ -0,0 +1,41 @@
+<!-- PASS -->
```

Comment:
Add pass and fail tests for all roles you're testing: `checkbox`, `combobox`, `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch`, `textbox`